docs/changelog.txt

```diff
@@ -111,6 +111,9 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - ``gui.Screen.show()`` now returns ``self`` as a convenience
 - ``gui.View.getMousePos()`` now takes an optional ``ViewRect`` parameter in case the caller wants to get the mouse pos relative to a rect that is not the frame_body (such as the frame_rect)
 
+## Build configuration
```

Comment:
this is failing because the list of valid headers is hard-coded elsewhere. I'd put this in `Internal`.

Reply:
oops sorry

Reply:
Meh, `Internals` not `Internal`. Third time's the charm we can hope